docs/eventing/brokers/broker-types/kafka-broker/configuring-kafka-features.md

```diff
@@ -0,0 +1,206 @@
+If you want to assign consumer group IDs to your triggers in a different way, you can set use any valid [go text/template](https://pkg.go.dev/text/template) text template
+to generate the consumer group IDs by setting `triggers.consumergroup.template: "your-text-template-here"` in the config map below.
+
+### Broker topic template
+
+By default, the topic created by a Kafka Broker is named from the template `{% raw %}knative-broker-{{ .Namespace }}-{{ .Name }}{% endraw %}`. If you want to change the template
+used to name Broker topics, you can set `brokers.topic.template: "your-text-template-here"`, where `"your-text-template-here"` is any valid [go text/template](https://pkg.go.dev/text/template)
+in the config map below.
+
+## Channel topic template
+
+By default, the topic created by a Kafka Channel is named from the template `{% raw %}messaging-kafka.{{ .Namespace }}.{{ .Name }}"`. If you want to change the template
+used to name Channel topics, you can set `channels.topic.template: "your-text-template-here"`, where `"your-text-template-here"` is any valid [go text/template](https://pkg.go.dev/text/template)
+in the config map below.
+
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: config-kafka-features
+    namespace: knative-eventing
+    data:
+        # Controls whether the dispatcher should use the rate limiter based on the number of virtual replicas.
+        # 1. Enabled: The rate limiter is applied.
+        # 2. Disabled: The rate limiter is not applied.
+        dispatcher.rate-limiter: "disabled"
+        
+        # Controls whether the dispatcher should record additional metrics.
+        # 1. Enabled: The metrics are recorded.
+        # 2. Disabled: The metrics are not recorded.
+        dispatcher.ordered-executor-metrics: "disabled"
+        
+        # Controls whether the controller should autoscale consumer resources with KEDA
+        # 1. Enabled: KEDA autoscaling of consumers will be setup.
+        # 2. Disabled: KEDA autoscaling of consumers will not be setup.
+        controller.autoscaler: "disabled"{% raw %}
+        
+        # The Go text/template used to generate consumergroup ID for triggers.
+        # The template can reference the trigger Kubernetes metadata only.
+        triggers.consumergroup.template: "knative-trigger-{{ .Namespace }}-{{ .Name }}"
+        
+        # The Go text/template used to generate topics for Brokers.
+        # The template can reference the broker Kubernetes metadata only.
+        brokers.topic.template: "knative-broker-{{ .Namespace }}-{{ .Name }}"
+        
+        # The Go text/template used to generate topics for Channels.
+        # The template can reference the channel Kubernetes metadata only.
+        channels.topic.template: "messaging-kafka.{{ .Namespace }}.{{ .Name }}"
+        {% endraw %}
```

Comment:
I'm not sure why we have `% raw` and `% endraw ...`?

Reply:
the markdown flavour we are using interprets the `{{ .variable }}` syntax and tries to do variable replacement there. To avoid this and render our templates properly, we have to use `% raw` and `% endraw `